### PR TITLE
MFTF: Fix failing CMS tests due to PageBuilder conflict

### PR DESCRIPTION
**Pull request by lbajsarowicz**
### Description (*)
Another day, another flaky tests introduced. Proper change in PageBuilder repository (related PR)

### Fixed Issues (if relevant)
Failing CI

### Manual testing scenarios (*)
Create PR and wait

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by lbajsarowicz** [comment]
@magento run all tests

Final, working version of fix.

image

**Comment by lbajsarowicz** [comment]
@magento run all tests
Added missing copyright :-)

**Comment by lbajsarowicz** [comment]
@magento run all tests
I have no idea why it's not building.

**Comment by lbajsarowicz** [comment]
@magento run all tests
Looks like I've had outdated `develop` branch for PageBuilder

**Comment by lbajsarowicz** [comment]
@magento run all tests
I'm forcing a restart. That takes too much time.

**Comment by lbajsarowicz** [comment]
@magento run all tests
Failures are not related with the scope of PR

**Comment by lbajsarowicz** [comment]
@magento run all tests
Updated branch to restart tests

**Comment by lenaorobei** [comment]
@magento run all tests

**Comment by ihor-sviziev** [comment]
@magento run all tests

**Review by lenaorobei — DISMISSED** [review]

**Comment by magento-engcom-team** [comment]
Hi @lenaorobei, thank you for the review.
[ENGCOM-7256](https://jira.corp.magento.com/browse/ENGCOM-7256) has been created to process this Pull Request

**Review by ihor-sviziev — DISMISSED** [review]

**Comment by magento-engcom-team** [comment]
Hi @ihor-sviziev, thank you for the review.
[ENGCOM-7256](https://jira.corp.magento.com/browse/ENGCOM-7256) has been created to process this Pull Request

**Review by engcom-Alfa — CHANGES_REQUESTED** [review]
Hi @lbajsarowicz 
Could you resolve merge conflicts?
Thanks!

File: app/code/Magento/Catalog/Test/Mftf/Test/StoreFrontRecentlyViewedAtStoreViewLevelTest.xml [hunk 1]
```diff
@@ -126,9 +127,9 @@
         </actionGroup>
 
         <grabTextFrom selector="{{StoreFrontRecentlyViewedProductSection.ProductName('2')}}" stepKey="grabDontSeeHomeProduct3"/>
-         <assertNotContains stepKey="assertNotSeeProduct3">
-			<actualResult type="const">$grabDontSeeHomeProduct3</actualResult>
-			<expectedResult type="const">$$createSimpleProduct3.name$$</expectedResult>
+        <assertNotContains stepKey="assertNotSeeProduct3">
+            <actualResult type="const">$grabDontSeeHomeProduct3</actualResult>
+			      <expectedResult type="const">$$createSimpleProduct3.name$$</expectedResult>
```
Comment (ihor-sviziev) [inline app/code/Magento/Catalog/Test/Mftf/Test/StoreFrontRecentlyViewedAtStoreViewLevelTest.xml]: @lbajsarowicz Could you fix formatting also here?

**Review by ihor-sviziev — CHANGES_REQUESTED** [review]

**Comment by lbajsarowicz** [comment]
Of course. Not always Github editor is good solution.

**Review by ihor-sviziev — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @ihor-sviziev, thank you for the review.
[ENGCOM-7256](https://jira.corp.magento.com/browse/ENGCOM-7256) has been created to process this Pull Request

**Comment by engcom-Alfa** [comment]
:heavy_check_mark: QA Passed

**Comment by engcom-Echo** [comment]
@magento run Functional Tests B2B, Functional Tests CE

**Pull request merged**